### ISSUE TITLE
'Reset' rejected MT pages?

### ISSUE THREAD

**Issue by SamCCSI** [NRGI/resourcecontracts.org#308]
regarding our earlier discussion of the pages that have been "rejected" I assume there is no way other than to now 'reset' those pages and wait for another transcriber to redo them? Eg. Thunder Bird. If you have a way of reversing my decision to reject, that would be preferable, but interested in your thoughts either way, as we need to have these processed. 
http://resourcecontracts.elasticbeanstalk.com/mturk/1046

**Comment by anjesh** [comment]
Unfortunately we cannot undo the rejection from the rejection. Clicking "reject" means  the work has been rejected from the Amazon Mturk - which cannot be undone. But we could have that page status changed in our system directly from database. Worker's effort will get lost, i hope proper reason was provided during rejection. 

Do you want us to revert those rejection in the database? Reset will create new HIT and have to wait for another transcription, like you said.

**Comment by SamCCSI** [comment]
Yes please - the gravity of rejections has since been explained to us, so these will not be common.

If you revert the rejection, does that just mean that it is then open for a new person to transcribe? THat's what we want. (Sounds the same as me hitting 'reset' though?)

**Comment by anjesh** [comment]
Yes. It re-creates the task in the Mturl and open for the new worker.

**Comment by SamCCSI** [comment]
Perfect - thank you!

**Comment by anjesh** [comment]
I mis-read you word, revert is something you can't do, we will have to do that. Reset is something you do and creates new task.

**Comment by anjesh** [comment]
By revert, i meant manually updating the database to approve the rejected work in our system only. The worker will not be paid as their work is already rejected.

**Comment by SamCCSI** [comment]
So if we revert, do we have access to the rejected transcription?

On Tue, Sep 15, 2015 at 9:41 AM, Anjesh <notifications@github.com> wrote:

> By revert, i meant manually updating the database to approve the rejected
> work in our system only. The worker will not be paid as their work is
> already rejected.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NRGI/resourcecontracts.org/issues/308#issuecomment-140396467>
> .
>



-- 



*SAM SZOKE-BURKE*
*Legal Researcher**Columbia Center on Sustainable Investment*
Columbia Law School - The Earth Institute, Columbia University
Level 1, Warren Hall, 410 W116th St. New York, NY 10027
| (212) 854-2635 <212-854-2635> | (646) 630-4123 <646-630-4123>
| s.burke@columbia.edu
| www.ccsi.columbia.edu | @CCSI_Columbia <https://twitter.com/CCSI_Columbia>
 | @samszokeburke <http://twitter.com/samszokeburke>

**Comment by anjesh** [comment]
yes, we already have that in our database as you may see in this rejected task page http://resourcecontracts.elasticbeanstalk.com/mturk/1046/420

**Comment by SamCCSI** [comment]
Perfect. Many thanks Anjesh.

Please revert all rejected pages, so that our in house intern can manually
adjust them to make them presentable.

Where does he access those pages to make additional manual amendments? I
assume they appear on the 'review contract text
<http://resourcecontracts.elasticbeanstalk.com/contract/1046/review>' page
only after we have accepted and published all pages back into the database?

On Tue, Sep 15, 2015 at 9:43 AM, Anjesh <notifications@github.com> wrote:

> yes, we already have that in our database as you may see in this rejected
> task page http://resourcecontracts.elasticbeanstalk.com/mturk/1046/420
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NRGI/resourcecontracts.org/issues/308#issuecomment-140397339>
> .
>



-- 



*SAM SZOKE-BURKE*
*Legal Researcher**Columbia Center on Sustainable Investment*
Columbia Law School - The Earth Institute, Columbia University
Level 1, Warren Hall, 410 W116th St. New York, NY 10027
| (212) 854-2635 <212-854-2635> | (646) 630-4123 <646-630-4123>
| s.burke@columbia.edu
| www.ccsi.columbia.edu | @CCSI_Columbia <https://twitter.com/CCSI_Columbia>
 | @samszokeburke <http://twitter.com/samszokeburke>

**Comment by anjesh** [comment]
That's correct. It will be updated once all the pages are approved and "send to RC" button clicked - enabled when all the pages are approved. Will do that tomorrow and let you know once done.

**Comment by SamCCSI** [comment]
Thanks very much Anjesh. That sounds perfect.

On Tue, Sep 15, 2015 at 10:11 AM, Anjesh <notifications@github.com> wrote:

> That's correct. It will be updated once all the pages are approved and
> "send to RC" button clicked - enabled when all the pages are approved. Will
> do that tomorrow and let you know once done.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NRGI/resourcecontracts.org/issues/308#issuecomment-140405654>
> .
>



-- 



*SAM SZOKE-BURKE*
*Legal Researcher**Columbia Center on Sustainable Investment*
Columbia Law School - The Earth Institute, Columbia University
Level 1, Warren Hall, 410 W116th St. New York, NY 10027
| (212) 854-2635 <212-854-2635> | (646) 630-4123 <646-630-4123>
| s.burke@columbia.edu
| www.ccsi.columbia.edu | @CCSI_Columbia <https://twitter.com/CCSI_Columbia>
 | @samszokeburke <http://twitter.com/samszokeburke>

**Comment by anjesh** [comment]
The two pages are fixed and auto-approved from the backend.

**Comment by SamCCSI** [comment]
Hello, can this issue be reopened?

Can the following MT rejections please be 'reverted' by YI as @anjesh sets out above:

http://resourcecontracts.elasticbeanstalk.com/mturk/1044
http://resourcecontracts.elasticbeanstalk.com/mturk/107
http://resourcecontracts.elasticbeanstalk.com/mturk/1025

CC @anderspeders @anjesh 

Thanks,
Sam

**Comment by SamCCSI** [comment]
Today all rejected HITs cannot be reset. Here's one example: http://resourcecontracts.elasticbeanstalk.com/mturk/1506?status=1&approved=2

![screen shot 2016-05-25 at 11 52 48 am](https://cloud.githubusercontent.com/assets/12913915/15546815/6807e5f4-226f-11e6-9308-babc080bb1e7.png)

**Comment by manishgs** [comment]
We've shown proper error message when there is an issue while resetting HIT.
